packages/tsdocs/src/index.ts

```diff
@@ -4,19 +4,19 @@
 // License text available at https://opensource.org/licenses/MIT
 
 /**
- * @packageDocumentation
- *
  * The `@loopback/tsdocs` package is an internal module to generate
  * {@link https://github.com/Microsoft/tsdoc | tsdoc} based API docs
  * for `@loopback/*` packages within
  * {@link https://github.com/strongloop/loopback-next | loopback-next} monorepo
  * managed by {@link https://github.com/lerna/lerna | Lerna}.
  *
+ * @remarks
  * It's built on top of {@link https://api-extractor.com | Microsoft API Extractor}:
  *
  * - {@link https://github.com/Microsoft/web-build-tools/tree/master/apps/api-extractor | api-extractor}
  * - {@link https://github.com/Microsoft/web-build-tools/tree/master/apps/api-documenter | api-documenter}
  *
+ * @packageDocumentation
```

Comment:
@achrinza Can you explain why we need to add `@remarks` here and push down `@packageDocumentation` to the bottom?

Reply:
Docs above `@remarks` are [used as a one-liner summary](https://api-extractor.com/pages/tsdoc/tag_remarks/). Multi-line summaries don't seem to be supported, not even by the [Loopack API Docs](https://loopback.io/doc/en/lb4/apidocs.authentication.html#interfaces) (see: `UserService` interface)

`@packageDocumentation` is a modifier. It's at the bottom to follow the format used by the [docs](https://api-extractor.com/pages/tsdoc/tag_packagedocumentation/).

Reply:
Sounds good. My understanding is that `@packageDocumentation` can be anywhere in the first comment of the file.